pyproject.toml

```
@@ -29,7 +29,8 @@ classifiers = [
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
-  "Programming Language :: Python :: 3.11"
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12"
```

```suggestion
  "Programming Language :: Python :: 3.12",
```
We can add the trailing comma (this is toml, not json) to simplify the diff when we add Python 3.13.

---

sure, it wouldn't harm.